### DISCUSSION TITLE
build problem in docker

### DISCUSSION THREAD

**Discussion by jimingda** [robotology/community#618] [Q&A]
https://github.com/robotology/icub-basic-demos

When build above in the docker, i have the following problem, anyone can help me? Thanks

CMake Error at demoRedBall/CMakeLists.txt:9 (find_package):
  Could not find a configuration file for package "ICUB" that is compatible
  with requested version "2.0.0".

  The following configuration files were considered but not accepted:

    /usr/local/lib/cmake/ICUB/ICUBConfig.cmake, version: 1.27.0

**Comment by pattacini** [comment]
Hi @jimingda 

Which docker image are you using?

It's very likely that you're trying to build the latest version of `icub-basic-demos` into a docker image that is not up-to-date.
